voice-interface/src/main/java/com/derpgroup/derpwizard/voice/model/SsmlDocumentBuilder.java

```diff
@@ -172,26 +175,51 @@ public SsmlDocumentBuilder() {
    * @return the document, never null
    */
   public @NonNull SsmlDocument build() {
-    StringBuilder buffer = new StringBuilder("<speak>");
+    return new SsmlDocument(build(true));
+  }
+
+  /**
+   * Builds a String representing this without SSML tags.
+   *
+   * @return The raw text, never null
+   */
+  public @NonNull String getRawText() {
```

Comment:
This is good, but it reminds me of a question from the other day - how do we handle text that starts out in Ssml format?  We don't currently have a "addTextAlreadyInSsmlFormat" method on the builder, and even if we did, how would we return that raw?